### PR TITLE
Implement course-color cache instead of per call

### PR DESCRIPTION
**Pull request by TimvdLippe**
Use a cache instead of per-call. Saves a lot of server calculations.

Must use https://github.com/Mentoraat/plannings-tool-server/pull/12

### CODE REVIEW

File: app/elements/elements.html [hunk 1]
```diff
@@ -32,11 +32,6 @@
 <link rel="import" href="../bower_components/paper-input/paper-textarea.html">
 <link rel="import" href="../bower_components/paper-button/paper-button.html">
 
-<!-- Uncomment next block to enable Service Worker Support (2/2) -->
-
-<link rel="import" href="../bower_components/platinum-sw/platinum-sw-cache.html">
```
Comment (GijsWeterings) [inline app/elements/elements.html:37]: Why did you delete this?

Comment (TimvdLippe) [inline app/elements/elements.html:37]: Was not doing anything, and we also don't have it in bower. So if you would run `bower prune`, your index.html would throw 404's

Comment (GijsWeterings) [inline app/elements/elements.html:37]: Aah okay. We could use it for offline caching of the frontend, but realistically we'll need api connectivity anyway

**Comment by GijsWeterings** [comment]
LGTM

File: app/elements/elements.html (hunk 1 above)
Comment (TimvdLippe) [inline app/elements/elements.html:37]: Yes, for now we can remove it.
